### ISSUE TITLE
GHC 9.2 support is missing

### ISSUE THREAD

**Issue by Kleidukos** [stepcut/plugins#16]
Hi! I'd like to start using `plugins` for an application of my own.

I tried to depend on it with GHC 9.2 as my compiler, and the following message makes me think that `plugins` is still trying to look up the old module hierarchy:

```
src/System/Plugins/LoadTypes.hs:33:1: error:
    Could not find module ‘HscTypes’
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
33 | import HscTypes
   | ^^^^^^^^^^^^^^^
```

You're probably busy, so would you accept a PR to fix this?

**Comment by stepcut** [comment]
Yes. In fact, accepting PRs and uploading a new version to hackage is the only work I do on this package.

**Comment by Kleidukos** [comment]
Got it! Thanks for being reactive. :)

**Comment by tchoutri** [comment]
@stepcut Hi! @cgibbard and I have ported the package to the latest GHC releases. Due to the copious amount of CPP involved for (what are frankly ancient) versions of GHC, I wanted your permission to switch the support window to GHC 8.10 minimum. What do you think of this?

**Comment by Kleidukos** [comment]
@stepcut Hi! @cgibbard and I have ported the package to the latest GHC releases. Due to the copious amount of CPP involved for (what are frankly ancient) versions of GHC, I wanted your permission to switch the support window to GHC 8.10 minimum. What do you think of this?

**Comment by stepcut** [comment]
Given the limited number of users of this package, GHC 8.10 sounds reasonable. If someone really needs to use an older compiler, they can just use an older version of the plugin. I say that as someone who is still using GHC 8.6.5 :)

**Comment by bleedsa** [comment]
I'm still getting this error with ghc 9.0.2, has anyone figured out a fix?
